### ISSUE TITLE
pkg load broken on Octave 4.2 and 6.x: '__octave_link_register_doc__' undefined 

### ISSUE THREAD

**Issue by apjanke** [apjanke/octave-jsonstuff#2]
Looks like my PKG_ADD hack for loading the QHelp files only works on 4.4, because the `__octave_link_register_doc__` octave-link function it wasn't there in 4.2.

https://travis-ci.com/apjanke/octave-jsonstuff/jobs/172467974

```
$ octave --eval "pkg load jsonstuff; pkg unload jsonstuff"
octave: X11 DISPLAY environment variable not set
octave: disabling GUI features
error: '__octave_link_register_doc__' undefined near line 33 column 1
The command "octave --eval "pkg load jsonstuff; pkg unload jsonstuff"" exited with 1.
```

Conditionalize its use in PKG_ADD/PKG_DEL.

**Comment by apjanke** [comment]
Fixed in https://github.com/apjanke/octave-jsonstuff/commit/8b40887715e4e4918931cecf8b992a64b7498032: Now it'll only try registering QHelp files on Octave 4.4 and later.

**Comment by apjanke** [comment]
Re-opening: `__octave_link_register_doc__` has been renamed to `__event_manager_register_doc__` in Octave 6. Need to handle it there too.

**Comment by apjanke** [comment]
Fixed. Released in version 0.3.1.
